features/apply_with_allowed_accounts.feature

```diff
@@ -43,7 +43,7 @@ Feature: Apply command with allowed accounts
     And I run `stack_master apply us-east-1 myapp-db`
     And the output should contain all of these lines:
       | Account '11111111' is not an allowed account. Allowed accounts are ["22222222"].|
-    Then the exit status should be 0
+    Then the exit status should be 1
```

Comment:
This is an example of an incorrect Cucumber validation. This scenario was ensuring a status `0`, but because it was only testing via the `CLI` class it missed the binstub returning `1`.

Now the feature correctly checks the value return in the command invocation.

Note: this is not a change in StackMaster behaviour.